ui/src/stylesheets/admonitionblock.scss

```diff
@@ -8,31 +8,26 @@
 
 .admonitionblock,
 .alert {
-  border: 1px solid rgb(196, 196, 196);
-  border-radius: var(--border-radius);
-  font-family: var(--heading);
-  color: rgb(36, 36, 36);
-  line-height: 100%;
+  display: grid;
```

Comment:
It's not necessary to use `display: grid` to achieve the desired layout here. The `::before` element should have `display: block` instead.